presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,56 @@
+* Add check to disallow invoking SQL functions in SQL function body.
+* Add support for limiting the total number of buffers per optimized repartitioning operator. The limit can be set by the configuration property ``driver.max-page-partitioning-buffer-count``.
+* Add peak task total memory to query stats.
+* Add :func:`scale_qdigest` function to scale a ``qdigest`` to a new weight.
+* Add :func:`myanmar_font_encoding` and :func:`myanmar_normalize_unicode` to support working with Burmese text
+* Add support for :func:`ST_AsText` to accept Spherical Geographies.
+* Add support for :func:`ST_Centroid` to accept Spherical Geography Points and MultiPoints.
+
+Pinot Connector Changes
+_______________________
+* Add support for mapping Pinot ``BYTES`` data type to Presto ``VARBINARY`` type.
+* Add support for mapping Pinot time fields with days since epoch value to Presto ``DATE`` type via the system property ``pinot.infer-date-type-in-schema``.
+* Add support for mapping Pinot time fields with milliseconds since epoch value to Presto ``TIMESTAMP`` type via the system prpoerty ``pinot.infer-timestamp-type-in-schema``.
+* Add Pinot Field type in to column comment field shown as ``DIMENSION``, ``METRIC``, ``TIME``, ``DATETIME``, to provide more information.
+* Add support for pushing down distinct count query to Pinot on a best-effort basis.
+* Add support for new Pinot Routing Table APIs.
+
+Hive Connector Changes
+______________________
+* Fix a bug in Hive split calculation which affects Parquet reader in few corner cases.
+* Fix AWS client metric reporting when using S3 select.
+* Add AWS client retry pause time metrics to ``PrestoS3FileSystemStats``.
+* Add table property ``preferred_ordering_columns`` to support writing sorted files for unbucketed table.
+* Add native Parquet Writer for Presto.
+* Add support for impersonation access by using HMS delegation token.
+* Add support for multi-HMS instances load balancing and breakdown metrics by HMS hosts.
+* Add support in file status cache to cache all tables. This could be enabled by setting the configuration property ``hive.file-status-cache-tables`` to ``*``.
+* Add configuration property ``hive.orc-compression-codec`` to override ``hive.compression-codec`` for ORC and DWRF formats. If specified, ORC and DWRF files are compressed using this codec. RC, Parquet, and other files are compressed using hive.compression-codec.
+
+Druid Connector Changes
+_______________________
+* Fix druid connector segment scan.
+* Add support for query processing pushdown via the ``druid.compute-pushdown-enabled`` configuration property.
+* Add support for count distinct pushdown.
```

Comment:
nit: I think this isn't needed because all of the query processing pushdown is new.